csm_test_utils/files_rotation.py

```diff
@@ -46,7 +45,7 @@ def create_file(dd_input="/dev/urandom", base_file="/tmp/base_file.data", bs=120
     if not os.path.exists(base_file) or (round(time.time() - os.path.getmtime(base_file)) / 60) > 60:
         files = [i for i in os.listdir()]
         if files:
-            subprocess.call(['rm', '-r'] + files)
+            [os.remove(i) for i in files]
```

Comment:
There is no need to `if files` if using `for`

Reply:
really?

Reply:
I suppose it can be simplified to 
```python
for file in os.listdir():
    os.remove(file)
```